log/CHANGELOG.md

```diff
@@ -37,6 +37,10 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 * [#17194](https://github.com/cosmos/cosmos-sdk/pull/17194) Avoid repeating parse log level in filterFunc.
 
+### Bug Fixes
+
+* [#17205](https://github.com/cosmos/cosmos-sdk/pull/17205) Fix marshaling cometbft `log.Lazy` struct.
```

Comment:
Let's not make the changelog CometBFT related. Can it be more general?

Reply:
no problem

Reply:
Is it okay to describe like this?

> Fix types that do not implement the json.Marshaler interface.

Reply:
Yeah works!